scan-command/src/test/resources/test-resources/core-rules/rule_checkpanic.bal

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+function getResult() returns int|error => 1;
+
+// Noncompliant
```

Comment:
Move the comment closer to the actual statement.

Also, Non-compliant rather than Noncompliant.